### DISCUSSION TITLE
Can you integrate conductor with AWS? Like Lambda and ECS?

### DISCUSSION THREAD

**Discussion by roshni06** [Netflix/conductor#2890] [Q&A]
I am not finding any documentation for this? Has this been developed?

**Comment by apanicker-nflx** [comment]
Seamless integration with serverless platforms like AWS Lambda etc are currently being worked on. It is called out in the [roadmap](https://github.com/Netflix/conductor/wiki/Roadmap#usability). Will provide an update in a couple of weeks on the progress.
